shared/components/LocaleSuggestionBanner.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div v-if="isOpen" class="locale-suggestion-banner">
     <!-- eslint-disable-next-line vue/no-v-html -->
-    <p v-html="$t('locale-suggestion-banner-text')"></p>
+    <p v-html="$t('locale-suggestion-banner-text', {url: domainOrg})"></p>
```

Comment:
Ce n'est pas grave du tout, mais pour plus de clarté j'aurais bien mis le nom de variable `domainOrgUrl` : 

```suggestion
    <p v-html="$t('locale-suggestion-banner-text', {domainOrgUrl: domainOrg})"></p>
```